spring-aot-test/src/test/java/org/springframework/aot/test/context/bootstrap/generator/TestContextAotProcessorTests.java

```diff
@@ -97,22 +98,41 @@ void processWritesContextLoaderMappingAtStandardLocation() {
 						"}");
 	}
 
+	@Test
+	void processWritesContextInitializersMapping() {
+		ContextBootstrapStructure structure = this.tester.process(
+				SampleApplicationTests.class, SampleApplicationAnotherTests.class, SimpleSpringTests.class);
+		assertThat(structure).contextBootstrapInitializer("TestContextBootstrapInitializer")
+				.removeIndent(1).lines().containsSubsequence(
+						"public static Map<String, Class<? extends ApplicationContextInitializer<?>>> getContextInitializers(",
+						"    ) {",
+						"  Map<String, Class<? extends ApplicationContextInitializer<?>>> map = new HashMap<>();",
+						"  map.put(\"org.springframework.aot.test.samples.app.SampleApplicationTests\", TestContextBootstrapInitializer0.class);",
+						"  map.put(\"org.springframework.aot.test.samples.app.SampleApplicationAnotherTests\", TestContextBootstrapInitializer0.class);",
+						"  map.put(\"org.springframework.aot.test.samples.simple.SimpleSpringTests\", SimpleSpringTestsContextInitializer.class);",
+						"  return map;",
+						"}");
+	}
+
 	@Test
 	void processInvokeTestNativeConfigurationRegistrar() {
 		ContextBootstrapStructure structure = this.tester.process(SampleApplicationTests.class);
 		assertThat(structure).hasResourcePattern("org/springframework/aot/test/samples/app/SampleApplication.class");
 	}
 
 	@Test
-	void processRegisterReflectionForContextLoadersMappingMethod() {
+	void processReflectionRegistrationForMappingMethods() {
 		ContextBootstrapStructure structure = this.tester.process(SampleApplicationTests.class);
 		assertThat(structure).hasClassDescriptor("com.example.TestContextBootstrapInitializer", (descriptor) -> {
-			assertThat(descriptor.getMethods()).singleElement().satisfies((methodDescriptor) -> {
-				assertThat(methodDescriptor.getName()).isEqualTo("getContextLoaders");
-				assertThat(methodDescriptor.getParameterTypes()).isEmpty();
-			});
 			assertThat(descriptor.getAccess()).isNull();
 			assertThat(descriptor.getFields()).isNull();
+			assertThat(descriptor.getMethods()).hasSize(2);
+			assertThat(descriptor.getMethods()).allSatisfy((methodDescriptor) -> {
```

Comment:
I am not fan of the | assertion here. We usually use `anySatisfy` when there is more than one element (there are examples in the codebase).

Reply:
OK. I'll take a look at other usage of `anySatisfy` and see if I can apply that here.